extension/updateswh.js

```diff
@@ -142,7 +142,7 @@ function testupdateforge(url, forgespecs) {
 
 function setupGitHub(url, pattern, type) {
     var projecturl = pattern.exec(url)[0]; // this is the url of the project
-    var userproject = projecturl.replace(/^https?:\/\/github\.com\//, ""); // this is the user+project fragment
+    var userproject = new URL(projecturl).pathname.substring(1); // this is the user+project fragment
```

Comment:
Thanks for the URL class! Why we need the substring(1) here?

Reply:
`pathname` will include the first slash, so it's `/user/project`. `pathname.substring(1)` is `user/project`

Reply:
Ok, got it.